tutorials.csv

```diff
@@ -394,4 +394,26 @@ AWS-Lambda-PSGI,https://perladvent.org/2019/2019-12-17.html,Taking the Sleigh to
 Memoize-Expire,https://perladvent.org/2019/2019-12-19.html,Memories of Past Lives
 Test2-V0,https://perladvent.org/2019/2019-12-21.html,Test2: Test Harder (but easier)
 Test2-Compare-Base,https://perladvent.org/2019/2019-12-22.html,Comparing More
-App-yath,https://perladvent.org/2019/2019-12-23.html,Testing our Impatience
\ No newline at end of file
+App-yath,https://perladvent.org/2019/2019-12-23.html,Testing our Impatience
+Mite,https://perladvent.org/2022/2022-12-01.html,Silent Mite
+MooseX-Getopt,https://perladvent.org/2022/2022-12-02.html,MooseX::Getopt saves Christmas
+Pithub,https://perladvent.org/2022/2022-12-03.html,Santa is on GitHub
+App-CGIThis,https://perladvent.org/2022/2022-12-04.html,northpole.cgi
+Mango,https://perladvent.org/2022/2022-12-05.html,Catching dreams
+Util-H2O,https://perladvent.org/2022/2022-12-06.html,St. Nick's Reindeers Need H2O!
+state,https://perladvent.org/2022/2022-12-07.html,The Sleigh odometer
+pEFL,https://perladvent.org/2022/2022-12-08.html,A Perlmas Tree
+Value-Diff,https://perladvent.org/2022/2022-12-09.html,What did Santa forget?
+Feature-Compat-Try,https://perladvent.org/2022/2022-12-10.html,The Christmas Time Machine
+Try-ALRM,https://perladvent.org/2022/2022-12-11.html,Wake up! Time to open presents! 
+GraphViz2,https://perladvent.org/2022/2022-12-12.html,The Delivery Map
+all,https://perladvent.org/2022/2022-12-14.html,Let the Elves Import Your Packages
+Mojolicious-Plugin-RevealJS,https://perladvent.org/2022/2022-12-15.html,Create Professional Slideshows with Mojolicious::Plugin::RevealJS
+relocatable-perl,https://perladvent.org/2022/2022-12-16.html,Relocatable Perl
+Net-CIDR-Lite,https://perladvent.org/2022/2022-12-17.html,Naughty or Nice Networks
+MIME-Base64,https://perladvent.org/2022/2022-12-18.html,TWVycnkgQ2hyaXN0bWFzIDop
+CPAN-Audit,https://perladvent.org/2022/2022-12-19.html,Find known reported vulnerabilities in the modules you use
+Carton,https://perladvent.org/2022/2022-12-21.html,Lapland Packaging Department - Part 2
+Chart-Lines,https://perladvent.org/2022/2022-12-23.html,At the Present Factory
+Chart-Bars,https://perladvent.org/2022/2022-12-24.html,Looking at the Christmas Numbers
+Perl-Dist-APPerl,https://perladvent.org/2022/2022-12-25.html,How to wrap a camel?
```

Comment:
The interesting thing about this year, is that the calendar went to Dec 27. I think the module from Dec 27 is not actually on CPAN, but the one from the 26th is. So, it would be helpful if we could have some flexibility in the script that generates this list.

Also, thanks very much, @kjt299!